drivers/ieee802154/ieee802154_dw1000.c

```diff
@@ -767,11 +767,88 @@ static int dwt_filter(const struct device *dev,
 	return -ENOTSUP;
 }
 
-static int dwt_set_power(const struct device *dev, int16_t dbm)
+uint8_t dwt_encode_power_tx(int16_t target_mdb)
+{
+	uint8_t target_int_x2;
+	uint8_t coarse_steps;
+	uint8_t coarse_x2;
+	uint8_t fine_steps;
+	uint8_t coarse_reg;
+	uint8_t fine_reg;
+	uint8_t reg;
+
+	if (target_mdb < 0) {
+		return 0xC0;
+	} else if (target_mdb > DWT_TX_POWER_CTRL_MAX_MDB) {
+		return 0x1F;
+	}
```

Comment:
```

	if (target_mdb < 0) {
		return 0xC0;
	}

	if (target_mdb > DWT_TX_POWER_CTRL_MAX_MDB) {
		return 0x1F;
	}

Whatever 0xC0 or 0x1F is...
```

Reply:
I agree with @jfischer-no that introducing a well-named macro for the magic numbers would be very helpful here.

Reply:
Fixed